Week3Task2.py

```diff
@@ -0,0 +1,73 @@
+#QUESTION 2: Number In Sequence
+"""
+    Generates a sequence of numbers based on the formula: N1 = 8, Ni = Ni-1 + 7 for i > 1.
+    N1=8
+    N2=N1+7
+    N3=N2+7 еtс.
+    or
+    N1 = 8, Ni = Ni-1 + 7 for i > 1.
+    """
+
+def num_in_seq1(n):
+    if n == 1:
+        return 8
+
+    else:
+        return num_in_seq(n - 1) + 7
```

Comment:
Minus 3 mark.  This function should have returned a recursive call to its self but instead returns another making the final solution wrong. Fails test cases.
Should have been
```python
def num_in_seq1(n):
    if n == 1:
        return 8

    else:
        return num_in_seq1(n - 1) + 7
  ```

Reply:
9 / 12 marks